core/templates/dev/head/pages/exploration_editor/exploration_metadata_modal_directive.html

```diff
@@ -10,15 +10,10 @@
                             label-text="Add a title: what should your exploration be called?"
                             form-style="{'font-weight':'normal'}">
   </exploration-title-editor>
-  <form name="objectiveForm">
-    <p ng-if="requireObjectiveToBeSpecified"
-       ng-class="{'has-error': (objectiveForm.objectiveInput.$touched || objectiveHasBeenPreviouslyEdited) && explorationObjectiveService.displayed.length < 15}">
-      Add a goal: what does the exploration help people to do?
-      <input name="objectiveInput" type="text" class="form-control" ng-model="explorationObjectiveService.displayed"
-             placeholder="Learn how to ...">
-      <span class="help-block" ng-show="(objectiveForm.objectiveInput.$touched || objectiveHasBeenPreviouslyEdited) && explorationObjectiveService.displayed.length < 15" style="font-size: smaller; color: #a94442">Please provide a complete sentence that describes what this exploration is about.</span>
```

Comment:
If the length is < 15, shouldn't we display this error message? Otherwise it may not be obvious to the user what is wrong, and they may become frustrated.